<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTagsWindow</class>
 <widget class="QDialog" name="AddTagsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Tags</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="label_layout">
    <item>
     <widget class="QLabel" name="body_part_label">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Body Part</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="type_label">
      <property name="text">
       <string>Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="movement_label">
      <property name="text">
       <string>Movement</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="equipment_label">
      <property name="text">
       <string>Equipment</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="check_box_layout">
    <item>
     <layout class="QVBoxLayout" name="body_part_boxes"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="type_boxes"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="movement_boxes"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="equipment_boxes"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>160</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="button_layout">
    <item>
     <widget class="QPushButton" name="accept_button">
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
